examples/Consumer/Program.cs

```diff
@@ -153,7 +153,7 @@ public static void Run_ManualAssign(string brokerList, List<string> topics, Canc
             {
                 // the group.id property must be specified when creating a consumer, even 
                 // if you do not intend to use any consumer group functionality.
-                GroupId = "groupid-not-used-but-mandatory",
+                GroupId = Guid.NewGuid().ToString(),
```

Comment:
oh dear, i think i might have happened when i tried to move your PR to the 1.9.x branch rather than master, but there were too many issues, so i reverted. can you fix that?

Reply:
to be clear, targeting master is fine, we just don't want this change.

Reply:
got it.